### ISSUE TITLE
UpdaterAbstractCreate: using of FeatureAttributes

### ISSUE THREAD

**Issue by tonimueller** [LeMonADE-project/LeMonADE#128]
I want to pose the question if everyone agrees that  the UpdaterAbstractCreate should require the FeatureAttributes? 
Personally, I would think it is more general to remove the ```setTag()``` in the UpdaterAbstractCreate and, if required, then implement a inherited class where the ```setTag()``` functions is added to the particular functions.

**Comment by MartinWenge** [comment]
This is absolutely correct, I can remove these dependencies tonight. This will cause some of our tools to crash (function calls commiting a type and other functions expecting a type set in the follow up procedure) but keeping this in mind should be a resolvable problem.

**Comment by tonimueller** [comment]
Thanks, if you cannot do all, I take care on the remaining integration ;-)
